WhaleSpotting/react-app/src/api/apiClient.ts

```diff
@@ -56,3 +57,20 @@ async function getGetSettings(): Promise<any> {
         headers: !token ? {} : { "Authorization": `Bearer ${token}` }
     };
 }
+
+export async function getConfirmedSightings(search: SearchSightingRequestModel, pageNumber = 1, pageSize = 10): Promise<SightingApiModel[]> {
+    return await fetch(`api/sightings/search?
+        ${search.species ? "species=" + search.species : ""}
+        ${search.longitude ? "&longitude=" + search.longitude : ""}
+        ${search.latitude ? "&latitude=" + search.latitude : ""}
+        ${search.location ? "&location=" + search.location : ""}
+        ${search.sightedFrom ? "&sightedFrom=" + search.sightedFrom : ""}
+        ${search.sightedTo ? "&sightedTo=" + search.sightedTo : ""}
+        ${search.orcaType ? "&orcaType=" + search.orcaType : ""}
+        ${search.orcaPod ? "&orcaPod=" + search.orcaPod : ""}
+        ${search.confirmed ? "&confirmed=" + search.confirmed : ""}
+        ${pageNumber ? "&pageNumber=" + pageNumber : ""}
+        ${pageSize ? "&pageSize=" + pageSize : ""}
+    `, await getGetSettings())
```

Comment:
```suggestion
        ${pageSize ? "&pageSize=" + pageSize : ""}`,
    await getGetSettings())
```
Can we move the end `, to the line above

Reply:
done